workflow/rules/pbrun.smk

```diff
@@ -13,6 +13,9 @@ rule pbrun_deepvariant:
     output:
         vcf=temp("parabricks/pbrun_deepvariant/{sample}_{type}.vcf"),
     params:
+        cuda="CUDA_VISIBLE_DEVICES={}".format(os.getenv("CUDA_VISIBLE_DEVICES"))
+        if os.getenv("CUDA_VISIBLE_DEVICES") is not None
+        else "",
```

Comment:
Since this is re-used multiple times we should put it in common file as a function and assign it the cuda variable, will make it easier to maintain